src/styles/mimetypes.styl

```diff
@@ -0,0 +1,11 @@
+/*
+Generates a list of rules like:
+.my-prefix-audio
+    background-image embedurl('path/to/icon-type-audio.svg')
+
+It's a quick way to generate all the selectors for the different file types
+*/
+mime-types(selector-prefix, icons-path)
```

Comment:
Aw shit you're right. I renamed `binary` to `bin` but I need to do the same with `spreadsheet`.